docs/user/03-30-management-of-service-instances-and-bindings.md

```diff
@@ -0,0 +1,189 @@
+# Management of the Service Instances and Service Bindings Lifecycle
+
+Use the SAP BTP Operator module to manage the lifecycle of service instances and service bindings in the Kyma environment.
+
+## Create a Service Instance
+
+1.  To create an instance of an SAP BTP service, create a ServiceInstance custom resource (CR) file following this example:
+
+    ```yaml
+        kubectl create -f - <<EOF 
+        apiVersion: services.cloud.sap.com/v1
+        kind: ServiceInstance
+        metadata:
+            name: {SERVICE_INSTANCE_NAME}
+            namespace: {NAMESPACE} 
+        spec:
+            serviceOfferingName: {SERVICE_OFFERING_NAME}
+            servicePlanName: {SERVICE_PLAN_NAME}
+            externalName: {SERVICE_INSTANCE_NAME}
+            parameters:
+              key1: val1
+              key2: val2
+        EOF
+    ```
+      In the **serviceOfferingName** and  **servicePlanName** fields, enter the name of the SAP BTP service you want to use and the service's plan respectively.
```

Comment:
I thought [service plan](https://help.sap.com/docs/btp/best-practices/basic-platform-concepts?locale=en-US&version=Cloud&q=service%20plan#loio363f0f68f9704830ac65c87a2562559b) is a kind of fixed term in BTP. It's probably now wrong to write "the service's plan" instead, but I wonder if we should stick to the common phrasing. However, I think you're more familiar with the Service Marketplace than I am, so feel free to disregard this suggestion.